src/main/msp/msp_protocol_v2_betaflight.h

```diff
@@ -30,3 +30,5 @@
 // MSP2_SET_TEXT and MSP2_GET_TEXT variable types
 #define MSP2TEXT_PILOT_NAME                      1
 #define MSP2TEXT_CRAFT_NAME                      2
+#define MSP2TEXT_PID_PROFILE_NAME                3
+#define MSP2TEXT_RATE_PROFILE_NAME               4
```

Comment:
Looks good but this should be defined in `MSPHelper`.
Maybe you can rework it from here: https://github.com/betaflight/betaflight-configurator/pull/3096